Mathlib/Lean/addLocalVarInfoForBinderIdent.lean

```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2022 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+
```

Comment:
Could this file, and `getBinderName.lean`, be combined into `Mathlib/Lean/Expr.lean`?

Reply:
Done